pkg/test/docker.go

```diff
@@ -17,13 +18,19 @@ type portBinding map[string]string
 
 type portMapping map[string]*int
 
+type hostMapping struct {
+	dialPort *int
+	setHost  *string
```

Comment:
Why not simply `host` and `port`?

Reply:
I just wanted to be very explicit about the purpose of those pointers:

- dialPort - is a port we are going to use to do a tcp dial to check if a host is reacheable
- setHost - is a pointer where we are going to set a host, once we can reach it

Reply:
hmm ok, not sure how much i like it, but if we at some point decide to change it its not going to change the api - so fine with me for the moment